packages/app/src/containers/ColorSwatchGrid.tsx

```diff
@@ -0,0 +1,38 @@
+
+import styled from 'styled-components';
+
+const ScrollContainer = styled.div`
```

Comment:
Handling custom scroll can look nice, but I find it more accessible to use the built-in scrollbar. It also handles theming and different operating systems automatically

Reply:
using the built-in scrollbar.